internal/version/version.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I run `golangci-lint` including gofmt and other lint tools.

Reply:
How come we didn't pick this up in the previous PRs? 🤔

Reply:
This change was not triggered by any lint issues, just me mistakenly pasting license with a tab instead of spaces. Will revert that back to how it was before.